src/google-optimize.js

```diff
@@ -0,0 +1,24 @@
+import { fs, registerFsReady } from './utils/fs';
+
+// make sure dataLayer is available
+window.dataLayer = window.dataLayer || [];
+
+// make sure gtag function is available
+window.gtag = window.gtag || function() {
+  dataLayer.push(arguments);
+};
+
+registerFsReady(
+  () => {
+    window.gtag('event', 'optimize.callback', {
+      callback: optimizeCallback
+    })
+  });
+
+// take the optimize callback and turn it into an FS event
+function optimizeCallback( value, name ) {
+  fs( "event" )( "Experiment Viewed", {
```

Comment:
Can we add one comment here that "Experiment Viewed" has a higher rate limit; hence, why it has been chosen as the event name.

Reply:
Good call.  Added that in there!